quickstart/src/index.js

```diff
@@ -7,32 +7,50 @@ var previewTracks;
 var identity;
 var roomName;
 
-// Attach the Tracks to the DOM.
-function attachTracks(tracks, container) {
-  tracks.forEach(function(track) {
-    container.appendChild(track.attach());
-  });
+
+// Attach given track to the DOM.
+function attachTrack(container, track) {
+  container.appendChild(track.attach());
 }
 
-// Attach the Participant's Tracks to the DOM.
-function attachParticipantTracks(participant, container) {
-  var tracks = getTracks(participant);
-  attachTracks(tracks, container);
+// Attach array of Tracks to the DOM.
+function attachTracks(container, tracks) {
+  tracks.forEach(track => attachTrack(container, track));
 }
 
-// Detach the Tracks from the DOM.
-function detachTracks(tracks) {
-  tracks.forEach(function(track) {
-    track.detach().forEach(function(detachedElement) {
-      detachedElement.remove();
-    });
-  });
+// Detach given track from the DOM
+function detachTrack(track) {
+  track.detach().forEach(element => element.remove());
+}
+
+function trackPublished(container, publication) {
+  if (publication.isSubscribed) {
+    attachTrack(container, publication.track);
+  }
+  publication.on('subscribed', track => attachTrack(container, track));
+  publication.on('unsubscribed', detachTrack);
+}
+
+function trackUnpublished(publication) {
+  if (publication.isSubscribed) {
+    detachTrack(publication.track);
+  }
+}
+
+// Attach the Participant's Tracks to the DOM.
+function onRemoteParticipantConnected(participant, container) {
+  // Attach any existing tracks published to DOM
```

Comment:
@makarandp0 Nitpick: I don't think this comment is needed. You already have a comment above `function trackPublished() {...}`.